packages/app-headless-cms/src/admin/components/ContentModelForm/ContentModelFormRender/fields/DateTimeWithoutTimezone.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import Input from "./Input";
+import { Grid, Cell } from "@webiny/ui/Grid";
+
+const DEFAULT_TIME = "00:00:00"
+const DEFAULT_DATE = "";
+
+const DateTimeWithoutTimezone = (props) => {
+  // "2020-05-18 09:00:00"
+  const [date, setDate] = React.useState("")
+  const [time, setTime] = React.useState("")
+  React.useEffect(() => {
+    if (props.bind.value === null) {
```

Comment:
This can be moved directly into the `useState`, which can take a function as its first argument.